Readme.md

```diff
@@ -2,8 +2,10 @@
 
 ## Requirements
 
-- yarn (`v1.22`)
-- node 16 (`lts/gallium`)
+- [Yarn](https://classic.yarnpkg.com/lang/en/docs/install) (`v1.22`)
+- node 16 (`lts/gallium`) (optional: [NVM](https://github.com/nvm-sh/nvm#installing-and-updating))
+- [Docker](https://docs.docker.com/get-docker/)
+- [Make](https://www.gnu.org/software/make/)
 
 ## Steps to Set Up Development Environment
 
```

Comment:
Can we please search/replace case-sensitive so that this is always capitalized, i.e, `MERMAID`?